skimage/measure/tests/test_regionprops.py

```diff
@@ -754,47 +835,6 @@ def test_moments_weighted_central():
     np.set_printoptions(precision=10)
     assert_array_almost_equal(wmu, ref)
 
-    # Verify test function
-    centralMpq = get_central_moment_function(INTENSITY_SAMPLE, spacing=(1, 1))
-    assert_almost_equal(centralMpq(0, 0), ref[0, 0])
-    assert_almost_equal(centralMpq(0, 1), ref[0, 1])
-    assert_almost_equal(centralMpq(0, 2), ref[0, 2])
-    assert_almost_equal(centralMpq(0, 3), ref[0, 3])
-    assert_almost_equal(centralMpq(1, 0), ref[1, 0])
-    assert_almost_equal(centralMpq(1, 1), ref[1, 1])
-    assert_almost_equal(centralMpq(1, 2), ref[1, 2])
-    assert_almost_equal(centralMpq(1, 3), ref[1, 3])
-    assert_almost_equal(centralMpq(2, 0), ref[2, 0])
-    assert_almost_equal(centralMpq(2, 1), ref[2, 1])
-    assert_almost_equal(centralMpq(2, 2), ref[2, 2])
-    assert_almost_equal(centralMpq(2, 3), ref[2, 3])
-    assert_almost_equal(centralMpq(3, 0), ref[3, 0])
-    assert_almost_equal(centralMpq(3, 1), ref[3, 1])
-    assert_almost_equal(centralMpq(3, 2), ref[3, 2])
-    assert_almost_equal(centralMpq(3, 3), ref[3, 3])
-
-    # Test spacing
-    spacing = (3.2, 1.2)
-    wmu = regionprops(SAMPLE, intensity_image=INTENSITY_SAMPLE,
-                      spacing=spacing)[0].moments_weighted_central
-    centralMpq = get_central_moment_function(INTENSITY_SAMPLE, spacing=spacing)
-    assert_almost_equal(wmu[0, 0], centralMpq(0, 0))
-    assert_almost_equal(wmu[0, 1], centralMpq(0, 1))
-    assert_almost_equal(wmu[0, 2], centralMpq(0, 2))
-    assert_almost_equal(wmu[0, 3], centralMpq(0, 3))
-    assert_almost_equal(wmu[1, 0], centralMpq(1, 0))
-    assert_almost_equal(wmu[1, 1], centralMpq(1, 1))
-    assert_almost_equal(wmu[1, 2], centralMpq(1, 2))
-    assert_almost_equal(wmu[1, 3], centralMpq(1, 3))
-    assert_almost_equal(wmu[2, 0], centralMpq(2, 0))
-    assert_almost_equal(wmu[2, 1], centralMpq(2, 1))
-    assert_almost_equal(wmu[2, 2], centralMpq(2, 2))
-    assert_almost_equal(wmu[2, 3], centralMpq(2, 3))
-    assert_almost_equal(wmu[3, 0], centralMpq(3, 0))
-    assert_almost_equal(wmu[3, 1], centralMpq(3, 1))
-    assert_almost_equal(wmu[3, 2], centralMpq(3, 2))
-    assert_almost_equal(wmu[3, 3], centralMpq(3, 3))
```

Comment:
I'm not sure why these asserts were removed?

Reply:
I think I added them back in. I removed them because of the weird CircleCI. Was trying to diagnose.

Reply:
Okay, in that case I think it's definitely something we should still add back and investigate.